src/xclim/testing/utils.py

```diff
@@ -567,8 +568,8 @@ def _downloader(
     return _nimbus
 
 
-# FIXME: This function is soon to be deprecated.
 # idea copied from raven that it borrowed from xclim that borrowed it from xarray that was borrowed from Seaborn
+@deprecated(from_version="0.57.0", suggested="nimbus")
```

Comment:
I understand the idea of switching to a more modern tool, but why the need to remove this function ?

Could we simply rewrite it so:
```python3
def open_dataset(name, nimbus_kwargs, **xr_kwargs):
    return xr.open_dataset(nimbus(**nimbus_kwargs).fetch(name), **xr_kwargs) 
```
?  (ok, maybe with a bit more doc and  annotations, but you see what I mean)

Yes the function looks simple, but having a very convenient function identical to `xr.open_dataset` seems a bonus to me! It makes doc tests and examples much more readable. And testing is also made simpler.

Reply:
Yeah, alright. It's simple and elegant enough to remain useful. I can't fight you on this, haha.

I'll make the changes.

Reply:
For complete clarity, the new function will work as intended, so long as the string being passed to it is in the registry. Anything else will error out.

Reply:
For complete clarity, the new function will work as intended, so long as the string being passed to it is in the registry. Anything else will error out.